cvat/apps/engine/views.py

```diff
@@ -115,6 +116,9 @@
 
 _UPLOAD_PARSER_CLASSES = api_settings.DEFAULT_PARSER_CLASSES + [MultiPartParser]
 
+_DATA_CHECKSUM_HEADER_NAME = 'X-Checksum'
```

Comment:
How is this checksum supposed to be used?

Reply:
It can be used to check if the downloaded chunk has the same checksum as the one cached, if there are any locally cached ones.